server/server.go

```diff
@@ -935,6 +935,17 @@ func (server *Server) loadConfig(configurations types.Configurations, globalConf
 						log.Infof("Configured IP Whitelists: %s", frontend.WhitelistSourceRange)
 					}
 
+					if len(frontend.Redirect) > 0 {
+						proto := frontend.Redirect
```

Comment:
This may be a silly question, but what happens if the entrypoint is not named for the proto its using?

For example, if I named an entrypoint `secure` for `:443` and `insecure` for `:80`?

Reply:
That's a good question. I have no idea how to fix this properly tough. Maybe we could use this as a takeway for a further refacto? I guess the use-case is pretty rare